src/openforms/js/components/admin/form_design/FormLogic.js

```diff
@@ -364,6 +303,23 @@ const Rule = ({
             </div>
           ) : null}
 
+          {triggerFromStepIdentifier && !triggerFromStep && (
+            <MessageList
+              messages={[
+                {
+                  message: (
+                    <FormattedMessage
+                      description="Warning missing trigger step"
+                      defaultMessage="The selected trigger step could not be found in this form!
+                      Please change it by clicking on the cog icon here on the left."
+                    />
+                  ),
+                  level: 'warning',
+                },
+              ]}
+            />
+          )}
```

Comment:
Is there a way we can pass this in so that the cog icon is automatically expanded and the warning shown with the relevant field, instead of having this indirect message?